include/SFML/Audio/InputSoundFile.hpp

```diff
@@ -153,6 +153,10 @@ class SFML_AUDIO_API InputSoundFile : NonCopyable
     /// precision. If you need to jump to a given time, use the
     /// other overload.
     ///
+    /// Attention: Bear in mind that the sample offset takes,
```

Comment:
Less drama? :P Something like "The sample offset takes the number of channels into account. Offsets can be calculated like this: ....".
Also there's a typo in "caculated".